plugins/DevicesDetection/Reports/GetModel.php

```diff
@@ -21,12 +21,14 @@ protected function init()
         $this->name          = Piwik::translate('DevicesDetection_DeviceModel');
         $this->documentation = ''; // TODO
         $this->order = 2;
+        $this->hasGoalMetrics = true;
         $this->widgetTitle  = 'DevicesDetection_DeviceModel';
     }
 
     public function configureView(ViewDataTable $view)
     {
         $view->config->show_search = true;
+        $view->config->show_goals = true;
```

Comment:
Can you check if this is really needed? In theory the framework should set it automatically see https://github.com/piwik/piwik/blob/2.16.1/core/Plugin/ViewDataTable.php#L203 . Ideally we make live for developers super easy and they don't have to set it :) I just noticed we could even set `$this->hasGoalMetrics = true` automatically when the related dimension implements `onAnyGoalConversion`. Will create an issue for that :)